lib/unleash/constraint.rb

```diff
@@ -80,8 +88,8 @@ def self.on_valid_version(val1, val2)
 
     # This should be a private method but for some reason this fails on Ruby 2.5
     def validate_constraint_value_type(operator, value)
```

Comment:
We should consider renaming this method, as it no longer validates, but only checks validity. And keeping as is would be misleading. 

Can't come up with a decent name now, but can try to think about a suggestion later on. Possibly we can consider in-lining it too with a comment? 🤷‍♂️ I dunno. 

Open for naming to suggestions here.

Reply:
done